app/helpers/markets_helper.rb

```diff
@@ -0,0 +1,15 @@
+module MarketsHelper
+  def current_buy_price(iex_price)
+    usd_to_php = 50.06
+    convert_price = iex_price * usd_to_php
+    with_interest_total = convert_price + (convert_price * 0.05)
+    number_to_currency(with_interest_total, unit: 'PHP ')
+  end
+
+  def current_sell_price(iex_price)
+    usd_to_php = 50.06
+    convert_price = iex_price * usd_to_php
+    with_interest_total = convert_price - (convert_price * 0.05)
+    number_to_currency(with_interest_total, unit: 'PHP ')
```

Comment:
you can improve these helpers right away to stay DRY

example, I can just right a method that accepts a boolean value (true for buy, false for sell) and returns the total that will be used by number_to_currency rails helper

```
def calculate_total(bool)
    usd_to_php = 50.06
    convert_price = iex_price * usd_to_php
    with_interest_total = bool ? convert_price + (convert_price * 0.05) : convert_price - (convert_price * 0.05)
end
```

Reply:
Hello, oh yeah right! That's a good idea, thanks! 

Please check if this is okay: https://github.com/mumble-07/emporia-wallet-app/pull/13/commits/bbd5d4beec58f4f7d8eca2551063ac7b505e0b31